C-Plus-Plus/dp/.cph/.Equal_sum_partition.cpp_d130310b148dc436cb9052663ab3183a.prob

```diff
@@ -0,0 +1 @@
+{"name":"Local: Equal_sum_partition","url":"c:\\Users\\Nandini Tiwari\\Desktop\\Git-Demo\\NeoAlgo\\C-Plus-Plus\\dp\\Equal_sum_partition.cpp","tests":[{"id":1616080413179,"input":"","output":""}],"interactive":false,"memoryLimit":1024,"timeLimit":3000,"srcPath":"c:\\Users\\Nandini Tiwari\\Desktop\\Git-Demo\\NeoAlgo\\C-Plus-Plus\\dp\\Equal_sum_partition.cpp","group":"local","local":true}
```

Comment:
Remove this

Reply:
Hey @HarshCasper , I have made the required changes please check.